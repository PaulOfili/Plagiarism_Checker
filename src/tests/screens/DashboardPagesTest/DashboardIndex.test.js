import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import DashboardIndex from '../../../screens/DashboardPages';

jest.mock('react-redux', () => ({
    useDispatch: ()=> {},
    useSelector: () => ({
        auth: { 
            isLoggedIn: false
        }
    })
}));

Enzyme.configure({adapter: new Adapter() })

function setup() {
    
    const enzymeWrapper = shallow(<DashboardIndex />);

    return {
        enzymeWrapper
    }
}

describe('Screens', () => {
    describe('Dashboard Index', () => {
        xit('should match the snapshot', () => {
            const {enzymeWrapper} = setup()
            expect(enzymeWrapper.html()).toMatchSnapshot();
        })
    })
})