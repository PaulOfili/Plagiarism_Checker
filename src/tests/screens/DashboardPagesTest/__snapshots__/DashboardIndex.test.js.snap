// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens Dashboard Index should match the snapshot 1`] = `"<div>Welcome to your Dashboard page<div><form action=\\"test/logout\\" style=\\"display:flex\\" method=\\"get\\"><input type=\\"text\\" name=\\"client_id\\" value=\\"1234\\" readonly=\\"\\" hidden=\\"\\"/><input type=\\"text\\" name=\\"redirect_uri\\" value=\\"test\\" readonly=\\"\\" hidden=\\"\\"/><input type=\\"text\\" name=\\"response_type\\" value=\\"token\\" readonly=\\"\\" hidden=\\"\\"/><input type=\\"text\\" name=\\"scope\\" value=\\"profile\\" readonly=\\"\\" hidden=\\"\\"/><input type=\\"text\\" name=\\"state\\" value=\\"ck4e6iku40000dknb2rktfllv\\" readonly=\\"\\" hidden=\\"\\"/><button type=\\"submit\\" style=\\"margin:0 auto\\" class=\\"ant-btn header-link header-signin\\">Sign Out</button></form></div></div>"`;
